From 40d3be22f6e518e4354aa7c3d0278291fcbed32f Mon Sep 17 00:00:00 2001
From: Sumit Bose <sbose@redhat.com>
Date: Fri, 5 Jun 2020 17:06:58 +0200
Subject: [PATCH] delete: do not exit if keytab cannot be read

Reading the keytab is not required when deleting a host object in AD. It
is only needed in the case where the host was added with a manual set
NetBIOS name (--computer-name option) which does not match the short
hostname and no computer name was given at the delete-computer command
line.

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=1840752
---
 tools/computer.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/computer.c b/tools/computer.c
index 292c4d8..a90c4b2 100644
--- a/tools/computer.c
+++ b/tools/computer.c
@@ -952,8 +952,6 @@ adcli_tool_computer_delete (adcli_conn *conn,
 	if (res != ADCLI_SUCCESS) {
 		warnx ("couldn't lookup domain info from keytab: %s",
 		       adcli_get_last_error ());
-		adcli_enroll_unref (enroll);
-		return -res;
 	}
 
 	res = adcli_conn_connect (conn);
-- 
2.23.0

